src/_includes/footer.html

```diff
@@ -3,13 +3,7 @@
         <div class="grid grid-cols-1 md:grid-cols-4 gap-12">
             <!-- Logo Column -->
             <div class="flex flex-col items-start">
-                <img
-                    class="mb-4 opacity-80 hover:opacity-100 transition-opacity"
-                    src="{{ '/img/BSM.png' | url }}"
-                    alt="Blue Square Monster"
-                    width="54"
-                    height="64"
-                />
+                {% image "src/img/BSM.png", "Blue Square Monster", "54px", "footer-logo" %}
```

Comment:
The image looks way bigger. 

Find in the image the size it is supposed to have:

<img width="1145" height="213" alt="Image" src="https://github.com/user-attachments/assets/c49583b4-6016-4cce-858a-24752a5d025a" />

Find in this other image the size it actually has in your change:

<img width="1379" height="437" alt="Image" src="https://github.com/user-attachments/assets/19056996-22d8-4f5a-b3f0-feeae5b7d503" />

please fix it